filament/src/main/java/net/fabricmc/filament/task/mappingio/CheckMappingsTask.java

```diff
@@ -0,0 +1,137 @@
+package net.fabricmc.filament.task.mappingio;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import net.fabricmc.mappingio.MappedElementKind;
+import net.fabricmc.mappingio.MappingVisitor;
+
+import org.gradle.api.file.DirectoryProperty;
+import org.gradle.api.tasks.InputDirectory;
+import org.gradle.api.tasks.TaskAction;
+import org.jetbrains.annotations.Nullable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import net.fabricmc.filament.task.base.FilamentTask;
+import net.fabricmc.mappingio.MappingReader;
```

Comment:
Checkstyle is failing on these out-of-order imports.

Reply:
Fixed